src/core/Alert/Alert/Alert.baseStyles.ts

```diff
@@ -1,77 +1,128 @@
 import { css } from 'styled-components';
 import { SuomifiTheme } from '../../theme';
-import { font } from '../../theme/reset';
-import { baseAlertBaseStyles } from '../BaseAlert/BaseAlert.baseStyles';
+import { element, font } from '../../theme/reset';
 
 export const baseStyles = (theme: SuomifiTheme) => css`
-  ${baseAlertBaseStyles(theme)}
+  &.fi-alert {
+    ${element(theme)}
+    ${font(theme)('bodyTextSmall')}
+    width: 100%;
 
-  & .fi-alert_style-wrapper {
-    display: block;
```

Comment:
`display: block;`
was not used because of the css specificity that overruled it and it would break the layout.